code/modules/mob/mob.dm

```diff
@@ -852,31 +852,40 @@ var/next_mob_id = 0
 /mob/proc/update_health_hud()
 	return
 
-/mob/living/on_varedit(modified_var)
-	switch(modified_var)
+/mob/living/vv_edit_var(var_name, var_value)
+	switch(var_name)
+		if("stat")
+			if((stat == 2) && (var_value < 2))//Bringing the dead back to life
+				dead_mob_list -= src
+				living_mob_list += src
+			if((stat < 2) && (var_value == 2))//Kill he
+				living_mob_list -= src
+				dead_mob_list += src
+	. = ..()
+	switch(var_name)
 		if("weakened")
-			SetWeakened(weakened)
+			SetWeakened(var_value)
```

Comment:
I started this 2 months ago and it sat mostly unfinished for 2 months, the number of conflicts i had to fix was incredible that i just kept a bunch of the code when fixing them.

This is some of the code i did 2 months ago, color how ever was a conflict